package.json

```diff
@@ -15,6 +15,9 @@
   "main": "dist",
   "types": "dist",
   "homepage": "https://github.com/bahmutov/cypress-angular-unit-test#readme",
+  "dependencies": {
+    "debug": "4.1.1"
```

Comment:
since you added `debug` module here, can you also add a note to the README like https://github.com/bahmutov/cypress-react-unit-test#debugging